cli/cmd/check.go

```diff
@@ -222,7 +224,18 @@ func configureAndRunChecks(cmd *cobra.Command, wout io.Writer, werr io.Writer, s
 		os.Exit(1)
 	}
 
-	if !success || !extensionSuccess {
+	totalSuccess := success && extensionSuccess
+
+	if options.output == healthcheck.ExtensionShortOutput {
+		fmt.Fprintln(wout, "")
+		if !totalSuccess {
+			fmt.Fprintf(wout, "Status check results are %s\n", failStatus)
+		} else {
+			fmt.Fprintf(wout, "Status check results are %s\n", okStatus)
+		}
```

Comment:
We only want to print to print a newline if the checks failed. That way, a successful `linkerd check -o short` results in

```
$ linkerd check -o short
Status check results are √
```

And an unsuccessful run results in

```
failures

Status check results are ×
```

```suggestion
		if !totalSuccess {
			fmt.Fprintf(wout, "Status check results are %s\n", failStatus)
		} else {
		    fmt.Fprintln(wout, "")
			fmt.Fprintf(wout, "Status check results are %s\n", okStatus)
		}
```